src/basics/installextensions.md

```diff
@@ -15,34 +19,30 @@ werden und im Rahmen eines Release-Prozesses bereitgestellt werden. Darüberhina
 gibt es noch *Community-Extensions*, die einen experimentellen oder instabilen
 Status haben und kein Teil des offiziellen *Release*-Prozesses sind.
 
-Im Rahmen des Workshops werden wir (exemplarisch) das WPS-Modul installieren.
+Im Rahmen des Workshops werden wir exemplarisch das WPS-Modul installieren.
 [WPS](http://www.opengeospatial.org/standards/wps) steht für *Web Processing Service*
-und ist (wie WMS und WFS) ein Standard des OGC, in dem Regeln für das Anfragen
+und ist (wie WMS und WFS) ein [Standard des OGC](http://www.opengeospatial.org/standards/wps), in dem Regeln für das Anfragen
 und Antworten von (räumlichen) Prozessen, definiert sind.
 
-![GeoServer-Erweiterungen. Rot markiert ist die WPS-Erweiterung.](../assets/gs_extensions.png)
-
 ## Installieren der WPS-Erweiterung
 
+![GeoServer-Erweiterungen. Rot markiert ist die WPS-Erweiterung.](../assets/gs_extensions.png)
+
 Führen Sie die folgenden Schritte aus, um diese Erweiterung zu installieren:
 
 1. Stoppen Sie den Geoserver durch einen Doppelklick auf **Stop GeoServer** im
    Ordner **Web Services** auf Ihrem Desktop.
-2. Laden Sie die WPS-Erweiterung von hier {{ book.workshopUrl }}materials/geoserver-{{ book.geoServerVersion }}-wps-plugin.zip
+1. Laden Sie die WPS-Erweiterung von hier {{ book.workshopUrl }}materials/geoserver-{{ book.geoServerVersion }}-wps-plugin.zip
    (oder der offiziellen Release-Seite) herunter. Wählen Sie bitte *Datei speichern*
    und nicht *Öffnen mit*!
-3. Wechseln Sie im Terminal in das im vorigen [Abschnitt](./folderstructure.md)
+1. Wechseln Sie im Terminal in das im vorigen [Abschnitt](./folderstructure.md)
```

Comment:
In Github Markdown these numbers don't matter. Subsequent lines starting with `1.` are formatted as numbered list and the numeration will be set automatically.